openlibrary/plugins/openlibrary/js/goodreads_import.js

```diff
@@ -89,39 +101,61 @@ export function initGoodreadsImport() {
                                         edition_id: obj['key'],
                                         bookshelf_id: shelf_id
                                     },
-                                    datatype: 'json',
+                                    dataType: 'json',
                                     success: function () {
-                                        $(`[isbn=${value['ISBN']}]`).append('<td class="success-imported">Imported</td>')
-                                        $(`[isbn=${value['ISBN']}]`).removeClass('selected');
+                                        if (value['Date Read'] !== '') {
+                                            let date_read = value['Date Read'].split('/'); // Format: "YYYY/MM/DD"
+                                            $.ajax({
+                                                url: `${obj['works'][0].key}/check-ins`,
```

Comment:
Is it preferable to use
```
`${obj['works'][0].key}/bookshelvers_events.json`
```
?

Reply:
I'll change, and review the code again. There might be a way to clean-up a little bit that code.